versions.cfg

```diff
@@ -13,44 +13,45 @@ Products.BTreeFolder2 = 5.1
 Products.ZCatalog = 7.1
 Pygments = 2.18.0
 Record = 4.1
-Sphinx = 7.3.7
-alabaster = 0.7.16
-certifi = 2024.2.2
+Sphinx = 8.0.2
+alabaster = 1.0.0
+certifi = 2024.7.4
 charset-normalizer = 3.3.2
 collective.recipe.template = 2.2
 colorama = 0.4.6
-# sphinx-rtd-theme 2.0.0 requires docutils <21
-docutils = 0.20.1
+docutils = 0.21.2
 five.localsitemanager = 4.0
+furo = 2024.8.6
 idna = 3.7
 imagesize = 1.4.1
-importlib-metadata = 7.1.0
+importlib-metadata = 8.2.0
 # Required by zope.testbrowser on Python 3.13 because WebOb still uses cgi.
-legacy-cgi = 2.6
+legacy-cgi = 2.6.1
 mr.developer = 2.0.2
-packaging = 24.0
+packaging = 24.1
 plone.recipe.command = 1.1
-requests = 2.31.0
+requests = 2.32.3
 snowballstemmer = 2.2.0
-sphinx-rtd-theme = 2.0.0
-sphinxcontrib-applehelp = 1.0.8
-sphinxcontrib-devhelp = 1.0.6
-sphinxcontrib-htmlhelp = 2.0.5
-sphinxcontrib-jquery = 4.1
+sphinx-basic-ng = 1.0.0b2
+sphinxcontrib-applehelp = 2.0.0
+sphinxcontrib-devhelp = 2.0.0
+sphinxcontrib-htmlhelp = 2.1.0
 sphinxcontrib-jsmath = 1.0.1
-sphinxcontrib-qthelp = 1.0.7
-sphinxcontrib-serializinghtml = 1.1.10
+sphinxcontrib-qthelp = 2.0.0
+sphinxcontrib-serializinghtml = 2.0.0
 tempstorage = 6.0
 tomli = 2.0.1
-urllib3 = 2.2.1
+urllib3 = 2.2.2
 z3c.checkversions = 2.1
 zc.recipe.testrunner = 3.0
-zipp = 3.18.2
-zope.testrunner = 6.4
+zipp = 3.19.2
+zope.testrunner = 6.5
 
 [versions:python38]
 # alabaster >= 0.7.14 requires Python 3.9
 alabaster = 0.7.13
+# Sphinx <8 requires docutils <0.21
+docutils = 0.20.1
 # Required dependency on Python 3.8 only
 importlib-resources = 6.4.0
 # Sphinx >= 7.2 requires Python 3.9+
```

Comment:
This line and the next conflict with earlier lines.

Reply:
Not sure what you mean. This is a Python 3.8-specific pin so yes, it will be different from the higher Python versions that do support the latest packages.